nvflare/private/fed/app/simulator/simulator_worker.py

```diff
@@ -160,7 +161,9 @@ def _create_client(self, args, build_ctx, deploy_args):
 
         client.token = args.token
         self._set_client_status(client, deploy_args, args.simulator_root)
+        start = time.time()
         self._create_client_cell(client, args.root_url, args.parent_url)
```

Comment:
Is the parent_url still used, or it's empty?

Reply:
empty